src/main/java/edu/iis/powp/adapter/IPlotterDrawPanelControllerAdapter.java

```diff
@@ -34,8 +35,13 @@ public void drawTo(int x, int y)
         line.setEndCoordinates(x, y);
         
         setPosition(x, y);
+        
+		ApplicationWithDrawer.getDrawPanelController().drawLine(line);
```

Comment:
1. Constructor injection should be used (using a parameter). This is unnecessary direct coupling with ApplicationWithDrawer class.
2. Inheritance is not used (no super drawLine calls). Should be removed!